class.photon.php

```diff
@@ -422,9 +421,30 @@ public function filter_image_downsize( $image, $attachment_id, $size ) {
 			 * 	 @type $size Image size. Can be a string (name of the image size, e.g. full) or an integer.
 			 * }
 			 */
-			apply_filters( 'jetpack_photon_override_image_downsize', false, compact( 'image', 'attachment_id', 'size' ) )
-		)
+			false === apply_filters( 'jetpack_photon_admin_allow_image_downsize', false, compact( 'image', 'attachment_id', 'size' ) )
```

Comment:
Go sites actually run their own mini instances of Photon served from the main site's domain (e.g. `https://wpjobmanager.com/wp-content/uploads/2014/02/2014-02-27-at-22.36-1024x609.png?w=100`) and we [filter the Photon domain](https://github.com/Automattic/vip-go-mu-plugins/blob/master/a8c-files.php#L50-L54) accordingly. So the URLs we'd be saving for Go sites would be their own domain. So, we're basically using the Photon code but not the public Photon service :)

Definitely makes sense to have a clear warning there though.